youtube_dl/extractor/audiomack.py

```diff
@@ -9,20 +9,21 @@
 
 
 class AudiomackIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?audiomack\.com/song/(?P<id>[\w/-]+)'
+    _VALID_URL = r'https?://(?:www\.)?audiomack\.com/(song)/(?P<id>[\w/-]+)'
```

Comment:
Sorry, that was a typo.  Originally I was detecting whether it was song or album, before I broke album off into its own extractor.